draft-ietf-taps-arch.md

```diff
@@ -398,9 +398,9 @@ The diagram below provides a high-level view of the actions and events during th
 
 * Message Properties: Message Properties are used to specify details about Message transmission. They can be specified directly on individual Messages, or can be set on a Preconnection or Connection as defaults. These properties might only apply to how a Message is sent (such as how the transport will treat prioritization and reliability), but can also include properties that specific protocols encode and communicate to the Remote Endpoint. When receiving Messages, Message Properties can contain information about the received Message, such as metadata generated at the receiver and information signalled by the remote endpoint.
 
-* Send: The action to transmit a Message over a Connection to the remote system. The interface to Send MAY include Message Properties specific to how the Message content is to be sent. The status of the Send operation MUST be delivered back to the sending application in an event ({{events}}).
+* Send: The action to transmit a Message over a Connection to the remote system. The interface to Send can accept Message Properties specific to how the Message content is to be sent. The status of the Send operation is delivered back to the sending application in an event ({{events}}).
 
-* Receive: An action that indicates that the application is ready to asynchronously accept a Message over a Connection from a remote system, while the Message content itself will be delivered in an event ({{events}}). The interface to Receive MAY include Message Properties specific to the Message that is to be delivered to the application.
+* Receive: An action that indicates that the application is ready to asynchronously accept a Message over a Connection from a remote system, while the Message content itself will be delivered in an event ({{events}}). The interface to Receive can include Message Properties specific to the Message that is to be delivered to the application.
 
```

Comment:
Don't see this change as sufficient.

The idea of adding Send and Receive properties seem close to  encapsulating important TAPS architecture, but as written here seem more appropriate to the API.

I do think this IS something we ought to call-out as a fundamental aspect of any TAPS system.
A TAPS API is REQUIRED to accept Message Properties specific to how a Message content is to be sent, but the details of how these messages are used can be system-dependent. A TAPS API MAY SUPPLY Message Properties specific to how a Message content is to be received, but the details of how these messages are used can be system-dependent.

Reply:
let's pick this up in/after 107.